include/faker-cxx/System.h

```diff
@@ -46,6 +46,19 @@ class System
      */
     static std::string fileExt(const std::optional<std::string>& mimeType = std::nullopt);
```

Comment:
please delete that function

Reply:
Deleting that function will break commonFileExt function. The function and enum I wrote only covers the different file types(audio, video, etc.). One solution is to enumerate all the mime types also.

Reply:
Ok so please do that enumeration, I think it would be better to have one function for that fileExt with this opt param like described above. CommonFileExt should be the same but adjusted so types will work.

Reply:
Also, could you please rename "Ext" to extension in all functions in this file?